src/roamer.c

```diff
@@ -5,51 +5,74 @@
 #include "roamer.h"
 #include "constants/maps.h"
 
+// Despite having a variable to track it, the roamer is
+// hard-coded to only ever be in map group 0
+#define ROAMER_MAP_GROUP 0
+
 enum
 {
-    MAP_GRP = 0, // map group
-    MAP_NUM = 1, // map number
+    MAP_GRP, // map group
+    MAP_NUM, // map number
 };
 
+#define ROAMER (&gSaveBlock1Ptr->roamer)
 EWRAM_DATA static u8 sLocationHistory[3][2] = {0};
 EWRAM_DATA static u8 sRoamerLocation[2] = {0};
 
+#define __ MAP_NUM(UNDEFINED) // For empty spots in the location table
```

Comment:
`__` conflicts with a preproc directive representing a charcoded string with no terminator.

Reply:
Preproc handles this fine. It expects a left paren afterwards, otherwise it continues on. Did you mean this IDE support hack?
<https://github.com/pret/pokeemerald/blob/master/include/global.h#L25>